docs/nuxt.config.js

```diff
@@ -1,4 +1,4 @@
-import NuxtPress from '../src'
+import NuxtPress from '../distributions/nuxt-press/'
```

Comment:
Does this exists at all?

Reply:
It works for me, but probably better to use @nuxt/press as that should work after lerna link

Reply:
Only noticed now that that folder was ignored by a greedy line in .gitignore, sorry about that!